go/README.md

```diff
@@ -1 +1,177 @@
-# Go
```

Comment:
In general just to clean up the conventions a bit:

Go over the document to cleanup typo's (misspellings, replace "var's" by "variables", capitalize the first letter of sentences/headlines, etc.)